### PR TITLE
Edit profile

### PR DESCRIPTION
**Pull request by felixportuphy**
Edit profile and change password features ,assertions to be done

### CODE REVIEW

File: cypress-open.json [hunk 1]
```diff
@@ -2,15 +2,16 @@
   "viewportWidth": 1920,
   "viewportHeight": 1080,
   "browser": "chrome",
-  "pageLoadTimeout": 40000,
-  "defaultCommandTimeout": 4000,
+  "pageLoadTimeout": 100000,
+  "defaultCommandTimeout": 10000,
```
Comment (JerryHukpati) [inline cypress-open.json:6]: don't push changes you make for debugging your test. your might end up breaking other tests. revert all changes you made to the cypress-open.json file

File: cypress-open.json [hunk 2]
```diff
@@ -2,15 +2,16 @@
...
   "testFiles": ["tests/**/*.spec.js"],
   "video": false,
   "headless": false,
   "env": {
     "mode-run": false
   },
-  "retries": 2,
+  "retries": 0,
```
Comment (JerryHukpati) [inline cypress-open.json:13]: refer to earlier comment on cypress-open.json

File: cypress-open.json [hunk 3]
```diff
@@ -2,15 +2,16 @@
...
   "scrollBehavior": "center",
-  "chromeWebSecurity": false
+  "chromeWebSecurity": false,
+  "watchForFileChanges": false
```
Comment (JerryHukpati) [inline cypress-open.json:16]: remove

File: cypress/fixtures/amali-changepassword.data.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
+export function changePasswordData() {
+    return {
+
```
Comment (JerryHukpati) [inline cypress/fixtures/amali-changepassword.data.js:3]: remove empty line

File: cypress/fixtures/amali-changepassword.data.js [hunk 5]
```diff
@@ -0,0 +1,11 @@
+export function changePasswordData() {
+    return {
+     
+      email: 'aegistest3@imboate.com',
+      oldPassword:'aegistest3',
+      password: 'aegistest3',
+      passwordRepeat: 'aegistest3',
+
```
Comment (JerryHukpati) [inline cypress/fixtures/amali-changepassword.data.js:8]: remove empty line

File: cypress/integration/tests/AccountSettings/amali-001-change-password.spec.js [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+      cy.location().then((loc) => {
+        if (loc.pathname === '/account-settings') {
+          cy.logOut()
+        } else {
+          cy.reload()
+        }
+      })
+    })
+
+    it.only('Should  change user account password ', () => {
```
Comment (JerryHukpati) [inline cypress/integration/tests/AccountSettings/amali-001-change-password.spec.js:44]:
```suggestion
    it('Should  change user account password ', () => {
```

File: cypress/support/commands.js [hunk 7]
```diff
@@ -97,6 +97,7 @@ Cypress.Commands.add('continueWithNonExistingEmail', () => {
 })
 
 Cypress.Commands.add('gotoSignInPage', () => {
+
```
Comment (JerryHukpati) [inline cypress/support/commands.js:100]: remove empty line

**Review by JerryHukpati — CHANGES_REQUESTED** [review]
the edit profile test can be found in Linda's PR as well. Fix all the comment and sort out the edit profile duplicate test and create a new PR

File: README.md [hunk 8]
```diff
@@ -1,76 +1,39 @@
...
-Docs: `/docs/screenshots`
 
-## Videos
 
-Directory: `cypress\videos`
-Docs: `/docs/videos`
 
-# Cypress
+   
+
```
[the remark this answers is not in the document]
Comment (felixportuphy) [reply README.md:36]: Resolved

File: cypress-open.json (hunk 1 above)
Comment (felixportuphy) [reply cypress-open.json:6]: Resolved

File: cypress-open.json (hunk 2 above)
Comment (felixportuphy) [reply cypress-open.json:13]: Resolved

File: cypress-open.json (hunk 3 above)
Comment (felixportuphy) [reply cypress-open.json:16]: Resolved

File: cypress/fixtures/amali-changepassword.data.js (hunk 4 above)
Comment (felixportuphy) [reply cypress/fixtures/amali-changepassword.data.js:3]: Resolved

File: cypress/integration/tests/AccountSettings/amali-001-change-password.spec.js (hunk 6 above)
Comment (felixportuphy) [reply cypress/integration/tests/AccountSettings/amali-001-change-password.spec.js:44]: Resolved

File: cypress/support/commands.js (hunk 7 above)
Comment (felixportuphy) [reply cypress/support/commands.js:100]: Resolved

**Review by nx-jeremiahhukpati — CHANGES_REQUESTED** [review]
merge the login branch into dev and rebase your branch. Re-request the review for this PR after that. Accept changes from dev when doing the rebase. Let me know if any have issues doing the rebase.

**Pull request closed without merging**